From 23d88f70a34c8ea536589da3903925aed4ad5c80 Mon Sep 17 00:00:00 2001
Message-Id: <23d88f70a34c8ea536589da3903925aed4ad5c80.1354540818.git.minovotn@redhat.com>
In-Reply-To: <2c5d72974e08333d09e77c0027c065e512738973.1354540818.git.minovotn@redhat.com>
References: <2c5d72974e08333d09e77c0027c065e512738973.1354540818.git.minovotn@redhat.com>
From: Eduardo Habkost <ehabkost@redhat.com>
Date: Wed, 14 Nov 2012 18:46:53 +0100
Subject: [PATCH 2/3] target-i386: cpu: add CPUID_EXT_PCID constant

RH-Author: Eduardo Habkost <ehabkost@redhat.com>
Message-id: <1352918814-16583-3-git-send-email-ehabkost@redhat.com>
Patchwork-id: 44216
O-Subject: [RHEL6.4 qemu-kvm PATCH 2/3] target-i386: cpu: add CPUID_EXT_PCID constant
Bugzilla: 869214
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>
RH-Acked-by: Igor Mammedov <imammedo@redhat.com>
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>

Bugzilla: 869214
Upstream status: c8acc380be7c7bb489037c927a2a5c217535f864

This adds the "pcid" feature name to the feature name array, and a
CPUID_EXT_PCID define that can be used on CPU model definitions.

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 target-i386/cpu.h | 1 +
 1 file changed, 1 insertion(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 target-i386/cpu.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target-i386/cpu.h b/target-i386/cpu.h
index c7d9374..7415335 100644
--- a/target-i386/cpu.h
+++ b/target-i386/cpu.h
@@ -392,6 +392,7 @@
 #define CPUID_EXT_CX16     (1 << 13)
 #define CPUID_EXT_XTPR     (1 << 14)
 #define CPUID_EXT_PDCM     (1 << 15)
+#define CPUID_EXT_PCID     (1 << 17)
 #define CPUID_EXT_DCA      (1 << 18)
 #define CPUID_EXT_SSE41    (1 << 19)
 #define CPUID_EXT_SSE42    (1 << 20)
-- 
1.7.11.7

